src/lightning_app/storage/mount.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+from typing import List
+
+__MOUNT_IDENTIFIER__: str = "__mount__"
+__MOUNT_PROTOCOLS__: List[str] = ["s3://"]
+
+
+@dataclass
+class Mount:
```

Comment:
Can we add Mount to the Drive e2e tests ?

Reply:
I'll create a new e2e test. We don't want to give off the impression that a drive and a mount are the same thing.